File: src/ManualList.java
import java.util.Scanner;
public class ManualList {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the maximum:");
        int max = sc.nextInt();
        
    System.out.println("Odd and Evenn from 1 to " + max + ":");
        for (int i = 1; i <= max; i++) {
            if (i % 2 == 0) {
                System.out.println(i + " is Even");
            } else if (i % 5 == 0) {
                System.out.println(i + " is factor of five");
            } else {
                System.out.println(i + " is Odd");
             }
    }sc.close();
    }   

}

File: javascript/ManualList.js
function runOddEven() {
    const maxInput = prompt("Enter the max number:"); // replace readline with prompt
    const max = parseInt(maxInput);

    console.log(`Odd/Even numbers from 1 to ${max}:`);

    for (let i = 1; i <= max; i++) {
        if (i % 2 === 0) {
            console.log(`${i} is even`);
        } else if (i % 5 === 0) {
            console.log(`${i} is multiple of 5`);
        } else {
            console.log(`${i} is odd`);
        }
    }
}
